packages/field-base/src/clear-button-mixin.js

```diff
@@ -60,8 +62,26 @@ export const ClearButtonMixin = (superclass) =>
      * @protected
      */
     _onClearButtonClick(event) {
+      event.preventDefault();
+      this._onClearAction();
+    }
+
+    /**
+     * @param {Event} event
+     * @protected
+     */
+    _onClearButtonMouseDown(event) {
       event.preventDefault();
       this.inputElement.focus();
+    }
+
+    /**
+     * @param {Event} event
+     * @param {boolean} focusInputElement
+     * @protected
+     */
+    _onClearButtonTouchEnd(event) {
+      event.preventDefault();
```

Comment:
Please add tests to cover this line, as currently commenting it out does not cause tests to fail.

Reply:
Actually, I am wondering now if we need to call `preventDefault()` on `touchend`. Testing on my iPhone, if I comment this line, there's no `focusout` event when I tap the clear button. Can't test on an Android device now.

Reply:
Tested on Android Chrome and there's no focusout event on clear button tap with this line commented.

Reply:
I removed it since it seems to not be needed. Also tested the field inside a grid-pro and it doesn't get removed when clear button is clicked.

Reply:
Then we probably don't need a touchend listener at all.

Reply:
BTW, we added `touchend` listener to prevent `click` to these components in the past (for older iOS versions):

- https://github.com/vaadin/vaadin-upload/pull/258
- https://github.com/vaadin/vaadin-text-field/pull/209

Reply:
I found on web.dev that there is such a thing as [mouse event emulation](https://web.dev/mobile-touchandmouse/#supporting-mouse-and-touch-together) on touch devices. The mouse events will be prevented if you prevent the touch events. 

I wonder if it would make sense to remove touchend and add a "isTouch" check to mousedown to only focus the input if it's a desktop device?

Reply:
Refactored to use `isTouch` and removed the `touchend` listener.